tools/nnyeah/nnyeah/nnyeah.csproj

```
@@ -26,4 +34,25 @@
       <DependentUpon>Errors.resx</DependentUpon>
     </Compile>
   </ItemGroup>
+
+  <Target Name="IncludeLegacy" BeforeTargets="_GetPackageFiles">
+    <PropertyGroup>
+      <ProjectRoot>$(MSBuildProjectDirectory)/../../..</ProjectRoot>
+      <LegacyiOSRoot>$(ProjectRoot)/_ios-build/Library/Frameworks/Xamarin.iOS.framework/Versions/git/lib/mono</LegacyiOSRoot>
+      <LegacyMacRoot>$(ProjectRoot)/_mac-build/Library/Frameworks/Xamarin.Mac.framework/Versions/git/lib/mono</LegacyMacRoot>
+      <NetRoot>$(MSBuildProjectDirectory)/../../../_build</NetRoot>
+    </PropertyGroup>
+    <ItemGroup>
+      <Content Include="$(LegacyiOSRoot)/Xamarin.iOS/Xamarin.iOS.dll;$(LegacyMacRoot)/Xamarin.Mac/Xamarin.Mac.dll">
+        <Pack>true</Pack>
+        <PackagePath>content/legacy</PackagePath>
+        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      </Content>
+      <Content Include="$(NetRoot)/Microsoft.iOS.Ref/ref/net6.0/Microsoft.iOS.dll;$(NetRoot)/Microsoft.macOS.Ref/ref/net6.0/Microsoft.macOS.dll">

```

We should try to avoid hardcoding `net6.0`, I've used this elsewhere with success:

```suggestion
      <Content Include="$(NetRoot)/Microsoft.iOS.Ref/ref/net$(BundledNETCoreAppTargetFrameworkVersion)/Microsoft.iOS.dll;$(NetRoot)/Microsoft.macOS.Ref/ref/net$(BundledNETCoreAppTargetFrameworkVersion)/Microsoft.macOS.dll">

```

---

I'm not sure about this one, as I expect we'll need a NET7 folder at some point, and move this from _build to macios-binaries or something.

I think I'd rather the build break than silently start copying in NET7 binaries into the net6 directory. 

Thoughts?

---

We already have a `net7.0` branch, so a solution will be needed for next time I merge `main` into `net7.0` (next week probably).

If we plan on using macios-binaries, then IMHO we should do that right away.